drivers/base/bootvid/i386/pc/vga.c

```diff
@@ -78,7 +78,7 @@ ReadWriteMode(IN UCHAR Mode)
 }
 
 VOID
-PrepareForSetPixel(VOID)
+VidpPrepareForSetPixel(VOID)
```

Comment:
```suggestion
PrepareForSetPixel(VOID)
```
to emphasise it's a per-arch dependent global helper function and not a generic private bootvid helper.